modules/cpmso-failure-domains-provider.adoc

```diff
@@ -21,6 +21,11 @@ The control plane machine set concept of a failure domain is analogous to existi
 |X
 |link:https://cloud.google.com/compute/docs/regions-zones[zone]
 
+|Nutanix
+//link:https://portal.nutanix.com/page/documents/details?targetId=Web-Console-Guide-Prism-v6_1:arc-failure-modes-c.html[Availability domain]
```

Comment:
Reference for later

Reply:
Should another Jira be raised to ensure that this link gets added to the docs? 

I'm not a fan of comments as they can easily slip through the cracks as reminders.